likeminds-chat-reactnative-integration/ChatSX/LMChatProvider.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from "react";
```

Comment:
Why this file is duplicate, I have seen he same file in example as well?

Reply:
deleted from `example`